### DISCUSSION TITLE
Are Logs still unstable?

### DISCUSSION THREAD

**Discussion by jonmorgs** [open-telemetry/opentelemetry-dotnet#3599] [Q&A] [answered]
As mentioned in the OpenTelemetryProtocol exporter [README.md](https://github.com/open-telemetry/opentelemetry-dotnet/blob/main/src/OpenTelemetry.Exporter.OpenTelemetryProtocol/README.md#otlp-logs), logs are not marked as stable, and as such are provided in a separate nuget package.

However, it appears in the OpenTelemetry proto [README.md](https://github.com/open-telemetry/opentelemetry-proto#maturity-level), that logs are now marked as stable? Should the OpenTelemetryProtocol exporter readme be updated to reflect this, and should the separate `Logs` package be moved into the `OpenTelemetry.Exporter.OpenTelemetryProtocol` package?

Possibly related issue #[1811](https://github.com/open-telemetry/opentelemetry-dotnet/issues/1811)?

**Comment by cijothomas** [comment, marked as answer]
Both Log data model and proto are now marked stable. We are working towards making the OTLP Exporter stable as well in the coming months.

**Reply by TiagoJVO** [reply]
Hi theere @cijothomas!

One year after, so I believe it is a good time to ask 😄 

> We are working towards making the OTLP Exporter stable as well in the coming months.

Can you share or point me to where this work is progressing and what can we expect in terms of timeline?

**Reply by cijothomas** [reply]
https://github.com/open-telemetry/opentelemetry-dotnet/milestone/37 Its expected to be stable in next ~3 weeks.

**Reply by cijothomas** [reply]
this was released as stable last week.
